oeqCommon/.project

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is the project file intentionally included?

Reply:
ah no - thanks for the catch